.github/dependabot.yaml

```diff
@@ -81,6 +86,12 @@ updates:
       interval: 'weekly'
       day: 'wednesday'
     open-pull-requests-limit: 1
+    labels:
```

Comment:
The only `ci-` label that has any impact in this list now is `ci-all-qa-tests`. To get say the EKS flavor to run a comment of `/test eks-qa-e2e-tests` would be required. (AKS is not operational at the moment.) Another reliable flavor to add would be `/test openshift-penultimate-qa-e2e-tests` if dependabot can comment like that.
I will go through the list of labels and remove the `ci-` flavors that no longer have any function.

Reply:
I cleaned up labels but wasn't able to spot a way to instruct dependabot to also write comments on PR. Checked this https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file